corehq/apps/api/tests.py

```diff
@@ -966,12 +988,22 @@ def __init__(self, objs):
     def obj_get_list(self):
         return self.objs
 
+    def detail_uri_kwargs(self, bundle_or_obj):
+        return {
+            'pk': get_obj(bundle_or_obj).other_model_id
```

Comment:
Looks like this is `other_model_id`, whereas in the other places it's `other_model_ids` - is that intentional?  What do those represent, exactly?